Makefile

```diff
@@ -433,7 +433,7 @@ TINYCC_CFLAGS= -DTCC_LIBTCC1=\"\0\"
 # That means we can't compile for a newer microarchitecture than native
 # We compile for an older microarchitecture on x64 to ensure compatibility
 .PHONY: tinycc
-tinycc:
+tinycc: ## to compile for an older microarchitecture on x64 to ensure compatibility
```

Comment:
that's already on line 434

Reply:
Yes, it's already on line 434 !

The helper provide the essential information for a specific Makefile target !
From how the helper works '##' using a search on the current Makefile, it's necessary to copy/paste that there !

![Screenshot from 2022-08-02 12-51-30](https://user-images.githubusercontent.com/22576758/182357665-a4869cf1-1725-4bd8-8993-5fc85150809a.png)